autofit/non_linear/search/abstract_search.py

```diff
@@ -913,7 +913,22 @@ def perform_update(
             return samples
 
         if self.is_master:
-            self.paths.save_samples(samples=samples)
+
+            samples_for_csv = samples
+
+            samples_weight_threshold = conf.instance["output"]["samples_weight_threshold"]
+
+            if samples_weight_threshold is not None:
+                samples_for_csv = samples_for_csv.samples_above_weight_threshold_from(
+                    weight_threshold=samples_weight_threshold
+                )
+
+                logger.info(
+                    f"Samples with weight less than {samples_weight_threshold} removed from samples.csv and not used to "
+                    f"compute parameter estimates errors, etc."

```

Comment:
```suggestion
                    "compute parameter estimates errors, etc."

```
you don't need the f, as it is not an f-string :-)

Reply:
Thanks!